### PR TITLE
Custom scroll bar

### PR DESCRIPTION
**Pull request by Ram-dev27**
issue #29 
reviewer @Ngoakor12

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -771,6 +791,9 @@ button {
   display: flex;
   align-items: center;
   gap: 14px;
+  padding: 15px 30px;
+  background-color: #ffffff;
+  max-height: 70px;
```
Comment (Ngoakor12) [inline src/App.css:796]: Remove these lines they ruin the skill-name styles and are unrelated to this PR.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Almost there, just a few things:

In hsl colors the last two parameters; saturation and lightness are percentages, please add them to make sure everything behaves as it should.

File: src/App.css [hunk 2]
```diff
@@ -63,6 +63,26 @@ button:focus-visible {
...
 
+/* Scrollbar */
+::-webkit-scrollbar {
+  background-color: hsl(210, 13, 81);
+  width: 12px;
+}
+
+::-webkit-scrollbar:hover {
+  background-color: hsl(210, 13, 71);
+}
```
Comment (Ngoakor12) [inline src/App.css:74]: These hover style should be applied to `::-webkit-scrollbar-thumb` for hover to take effect.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 2 above)
Comment (Ram-dev27) [reply src/App.css]: @Ngoakor12  Thanks for the comments,  I have add the hover effect for scroll bar thumb, and removed the unrelated commit code in the App.css file. You can check now . if anything need to change please let me know !

**Comment by Ngoakor12** [comment]
I can't pull your changes on my machine because you deleted(unforked) the repo.

But the preview shows that the hover is not working.

Fix: add percentages to the last two parameters of hsl color values

**Comment by Ram-dev27** [comment]
@Ngoakor12 I have create the new pull request #57 for that issue please ignore this one

**Pull request closed without merging**